src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.tsx

```diff
@@ -67,24 +83,77 @@ export const ContactDetailsHeader: React.FC<Props> = ({
   contactDetailsLoaded,
   contextType,
 }: Props) => {
+  const { pathname } = useRouter();
   const { data } = useGetContactDetailsHeaderQuery({
-    variables: { accountListId, contactId },
+    variables: {
+      accountListId,
+      contactId,
+      // Don't show the duplicate banner on the merge contacts page
+      loadDuplicate:
+        pathname !==
+        '/accountLists/[accountListId]/tools/merge/contacts/[[...contactId]]',
+    },
   });
   const loading = !data;
   const { t } = useTranslation();
 
   const [editPartnershipModalOpen, setEditPartnershipModalOpen] =
     useState(false);
 
+  const [duplicateContactId, setDuplicateContactId] = useState<string | null>(
+    null,
+  );
+
   useEffect(() => {
     if (!loading && !contactDetailsLoaded) {
       setContactDetailsLoaded(true);
     }
     return () => setContactDetailsLoaded(false);
   }, [loading]);
 
+  // Populate duplicateContactId once the data loads
+  useEffect(() => {
+    const duplicate = data?.contactDuplicates?.nodes[0];
+    if (duplicate) {
+      setDuplicateContactId(
+        duplicate.recordOne.id === contactId
+          ? duplicate.recordTwo.id
+          : duplicate.recordOne.id,
+      );
+    } else {
+      setDuplicateContactId(null);
+    }
+  }, [data]);
+
   return (
     <Box sx={{ padding: 2, backgroundColor: 'transparent' }}>
+      {duplicateContactId && (
+        <DuplicateAlert
+          severity="warning"
+          onClose={() => {}}
+          action={
+            <>
+              <Link
+                href={`/accountLists/${accountListId}/tools/merge/contacts?duplicateId=${duplicateContactId}`}
+                passHref
+              >
+                <Button sx={{ m: 1 }} variant="contained" component="a">
+                  {t('See Match')}
+                </Button>
+              </Link>
+              <IconButton
+                size="large"
```

Comment:
I know in Angular the close button is huge. But maybe this could be the same size as other close buttons since it's a circle.

Reply:
Good point ✅